keyboards/anavi/macropad10/info.json

```
@@ -0,0 +1,63 @@
+{
+    "keyboard_name": "Macro Pad 10",
+    "manufacturer": "ANAVI",
+    "url": "https://github.com/AnaviTechnology/anavi-macro-pad-10",
+    "maintainer": "leon-anavi",
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "cols": ["GP6", "GP7", "GP0"],
+        "rows": ["GP29", "GP28", "GP27"]
+    },
+    "features": {
+        "bootmagic": false,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": false,
+        "nkro": true,
+	"rgblight": true
+    },
+    "rgblight": {
+        "pin": "GP3",
+        "led_count": 4,
+	"hue_steps": 10,
+	"saturation_steps": 17,
+        "brightness_steps": 17,
+	"max_brightness": 255,
+	"animations": {
+            "all": true
```

Could you explicitly define which animations are enabled? 
```suggestion
            "alternating": true,
            "breathing": true,
            "christmas": true,
            "knight": true,
            "rainbow_mood": true,
            "rainbow_swirl": true,
            "rgb_test": true,
            "snake": true,
            "static_gradient": true,
            "twinkle": true
```

---

Thank you for the feedback. I have modified `info.json` and forced pushed the updated commit. Thanks, Leon